pipeline_dp/mean_var_params.py

```diff
@@ -0,0 +1,123 @@
+"""DP computing of count, sum, mean, variance."""
+
+import numpy as np
+import pipeline_dp
+
+from dataclasses import dataclass
+
+
+@dataclass
+class MeanVarParams:
+	"""Manages the parameters used for computing the mean/variance."""
+	eps: float
+	delta: float
+	low: float
+	high: float
+	max_partitions_contributed: int
+	max_contributions_per_partition: int
+	noise_kind: pipeline_dp.NoiseKind  # Laplace or Gaussian
+
+
+def _apply_laplace_mechanism(value, dp_params):
+	"""Applies the Laplace mechanism.
+
+	Args:
+		value: The result of querying the database.
+		dp_params: The parameters used at computing the noise.
+	"""
+	l1_sensitivity = dp_params.max_partitions_contributed * dp_params.max_contributions_per_partition
+
+	# TODO: use the secure noise instead of np.random
+	return value + np.random.laplace(0, l1_sensitivity / dp_params.eps)
+
+
+def _apply_gaussian_mechanism(value, dp_params):
+	"""Applies the Gaussian mechanism.
+
+	Args:
+		value: The result of querying the database.
+		dp_params: The parameters used at computing the noise.
+	"""
+	l2_sensitivity = np.sqrt(dp_params.max_partitions_contributed) * dp_params.max_contributions_per_partition
+	sigma = np.sqrt(2 * np.log(1.25 / dp_params.delta)) * l2_sensitivity / dp_params.eps
```

Comment:
Please 
1.extract computing sigma to a separate function (it would be easier then to replace with another formula)
2.add a link to the source of this formula (i.e. link to the book and theorem number)